packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -37,6 +38,7 @@ export type MenuButtonSlotProps = {
   contextualMenu: React.PropsWithRef<ContextualMenuProps>;
   contextualMenuItems: Pick<MenuButtonProps, 'menuItems'>;
   contextualMenuItem: MenuButtonItemProps;
+  svg: SvgProps;
```

Comment:
could we give this slot a more descriptive name like chevron or endIcon?

Reply:
It's been renamed to chevronSvg